api/config.yaml

```diff
@@ -13,6 +13,8 @@ components:
     ConnectionResult:
       type: object
       properties:
+        endpointLinting:
```

Comment:
I'm not sure about this name. I'm accepting suggestions

Reply:
As we are only checking the port, maybe `portCheck` ? 🤔

Reply:
used `portCheck` because it's way better than `endpointLinting`